test/mpi/rma/testlist.def

```diff
@@ -1,13 +1,8 @@
 winname 2
 allocmem 2
-putfence1 2
 putfidx 4
-getfence1 2
-accfence1 4
```

Comment:
This test (and others) went from having no specific time limit to having a 900 second time limit. I'm confused. Is this supposed to make `accfence1` more comprehensive or break it into multiple tests that cover everything but have a shorter runtime individually?

Reply:
Yes, most of the test have a double for loop for send and recv types. For each signatures there are 41 types in the pool for a total of 41^2 combinations, many more than before.